config/database.php

```diff
@@ -70,6 +70,53 @@
             ]) : [],
         ],
 
+        'mysql_cluster' => [
+            'read' => [
+                'host' => [
+                    env('DB_HOST', 'localhost'),
+                    env('DB_HOST_R2', ''),
+                    env('DB_HOST_R3', ''),
+                    env('DB_HOST_R4', ''),
+                    env('DB_HOST_R5', ''),
+                    env('DB_HOST_R6', ''),
+                    env('DB_HOST_R7', ''),
+                    env('DB_HOST_R8', ''),
+                    env('DB_HOST_R9', ''),
+                ],
+            ],
+            'write' => [
+                'host' => [
+                    env('DB_HOST', 'localhost'),
+                    env('DB_HOST_W2', ''),
+                    env('DB_HOST_W3', ''),
+                    env('DB_HOST_W4', ''),
+                    env('DB_HOST_W5', ''),
+                    env('DB_HOST_W6', ''),
+                    env('DB_HOST_W7', ''),
+                    env('DB_HOST_W8', ''),
+                    env('DB_HOST_W9', ''),
+               ],
+            ],
+            'sticky' => env('DB_STICKY', null),
+            'driver' => 'mysql',
+            'host' => env('DB_HOST', 'localhost'),
```

Comment:
is this line needed?  Or some LibreNMS code depending on it by mistake?

Reply:
DB_Host is needed because things like lnms break without it, might be couple of other things aswell.